back/src/config/auth.config.ts

```diff
@@ -1,4 +1,5 @@
 export default {
 	secret: String(process.env.JWT_SECRET || 'SUp3r_D3f4u1t_53cr3t'),
```

Comment:
I remember we talked about this but could you explicit once more why is there this hardcoded 'SUp3r_D3f4u1t_53cr3t' it really looks like a vulnerability as is.

Reply:
Done.
I simply added comments to explain it's only default value, set only if the env variable is not defined, and the fact it must be defined on the .env file! 
Is it ok?

Reply:
perfect cheers!